### DISCUSSION TITLE
personal best sometimes mismatches with the global leaderboard place on a beatmap

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#25874] [Q&A]
## globally(lazer) I apparently have rank 17 on the map, the UI shows me a 17 next to my personal best, which used to be the global rank if im correct as in this picture

![image](https://github.com/ppy/osu/assets/87947227/d22c8f3d-0195-4602-9dd0-9a42b5f7e437)

## here the ui shows me a 2, even though I have rank 42.

![image](https://github.com/ppy/osu/assets/87947227/89d69f4f-bebd-45bb-ac5a-14001cb7cbe9)

**Comment by Stoppedpuma** [comment]
https://github.com/ppy/osu/issues/14710
